tests/integration/test_ols.py

```diff
@@ -97,3 +100,40 @@ def test_post_question_on_unknown_response_type() -> None:
         assert response.json() == {
             "detail": {"response": "Internal server error. Please try again."}
         }
+
+
+@patch("ols.constants.PRODUCT_DOCS_PERSIST_DIR", "./invalid_dir")
```

Comment:
when you have to do something like this, it points out an area where the existing code is insufficiently configurable.  After https://github.com/openshift/lightspeed-service/pull/166 merges, please make the RAG directory a configurable item and then update this test to drive the config.

Reply:
Will do. Thanks